mriqc/engine/plugin.py

```diff
@@ -339,7 +329,10 @@ def _task_finished_cb(self, jobid, cached=False):
         rowview = self.depidx.getrowview(jobid)
         rowview[rowview.nonzero()] = 0
         if jobid not in self.mapnodesubids:
-            self.refidx[self.refidx[:, jobid].nonzero()[0], jobid] = 0
+            try:
+                self.refidx[self.refidx[:, jobid].nonzero()[0], jobid] = 0
+            except NotImplementedError:
+                self.refidx[self.refidx[:, [jobid]].nonzero()[0], jobid] = 0
```

Comment:
what is this addressing? Is there an issue open for this failure condition?

Reply:
When I changed l. 351, MRIQC was running into a NotImplementedError because they did not implement the feature for 1D arrays. Sorry, I cannot find again the exact error message so I don't remember which software was issuing the error. But, the error message suggested this workaround.